### DISCUSSION TITLE
Citty Naming Themes or Available MOD

### DISCUSSION THREAD

**Discussion by JezzyBelle** [voxeltycoon/issues#848] [General]
SUGGESTION: City Naming Themes/Mod. Wouldn't it be nice to have a flat file (notepad) or some method similar where you could inject names the game can pull from for city naming? Only names you provide. OR have modded city name Themes. Personally, I had one map were I re-named every new city as early on as I could afford to a known Historical US mining town (Past and Present). IE: Virginia City, Butte, Cripple Creek.. etc. It was extermely cost prohibitive and ultimately contributed to my downfall. SO... IN addition to a Countries localization files, it would be nice to tell it what genre. Some may like Major cities, Tourist,.. etc. Like I say, personally, I would like it to only use old Mining town names...

**Comment by kalmix88-hun** [comment]
Custom city names. There could be an option like this when generating maps.
